core/jazz_t-vault/config/dev-config.json

```diff
@@ -0,0 +1,14 @@
+{
+  "T_VAULT_API": "{tvault-host-name}/vault/v2",
```

Comment:
Use `T-VAULT` instead of `T_VAULT` (represents that it's part of the same word).

Reply:
@devsatishm `-` in key of map is not considered good practice as it is can't be directly referenced eg. `map.T_VAULT`. Would you prefer `TVAULT_API` or `T_VAULT_API`?